crates/starknet-devnet/tests/test_simulate_transactions.rs

```diff
@@ -232,6 +234,204 @@ mod simulation_tests {
         );
     }
 
+    async fn update_gas_scenario(devnet: BackgroundDevnet, assert_chain_id: &str) {
+        // get account
+        let (signer, account_address) = devnet.get_first_predeployed_account().await;
+        let account = SingleOwnerAccount::new(
+            devnet.clone_provider(),
+            signer,
+            account_address,
+            FieldElement::from_hex_be(assert_chain_id).unwrap(),
+            ExecutionEncoding::New,
+        );
+
+        // get class
+        let (flattened_contract_artifact, casm_hash) =
+            get_flattened_sierra_contract_and_casm_hash(CAIRO_1_CONTRACT_PATH);
+
+        let max_fee = FieldElement::ZERO;
+        let nonce = FieldElement::from_hex_be("0x0").unwrap();
```

Comment:
Why not also use `FieldElement::ZERO`?

Reply:
Fixed in cd01a81adf4b7003ffb013b6b1f00fcac5ab97c4